beets/util/artresizer.py

```diff
@@ -121,10 +124,12 @@ def pil_getsize(path_in):
 
 
 def im_getsize(path_in):
-    cmd = ['identify', '-format', '%w %h',
-           util.syspath(path_in, prefix=False)]
+    cmds = (['magick', 'identify'], ['identify'])
+    cmd = cmds[0] if not ArtResizer.shared.im_legacy else cmds[1]
+    args = ['-format', '%w %h', util.syspath(path_in, prefix=False)]
```

Comment:
I'd probably stick with the name `cmd` for the thing that includes both the executable name and the arguments:

```py
cmd = (the command) + ['-format', '%w %h', ...]
```

Reply:
Added in 48be3a7